### DISCUSSION TITLE
Any news on Apple M1 AAX Plugin Support?

### DISCUSSION THREAD

**Discussion by brendanreza** [apohl79/audiogridder#687] [General]
Is this in the works?

**Comment by apohl79** [comment]
Nope. Last time I checked, the current AAX SDK was not building for the arm64 target platform and would have required manual patches. Avid has not released a new SDK version, but at least they are listening Pro Tools as being supported now. I will check, if the SDK compiles on the latest Xcode version at some point and if it does, provide AAX AG builds for M1s.

**Reply by Jarrodsz** [reply]
So m1 is not supported yet?
Thats the reason perhaps why my plugins do not show in pro tools.

Would it be possible to run intel version of pro tools on m1 and also the audiogridder to revert back to rosetta and be able to use audiogridder on m1 pro tools?

**Comment by brendanreza** [comment]
Ahh got it.  So it's more on them, not Audio Gridder.  Thanks for the info.

**Reply by Jarrodsz** [reply]
So m1 is not supported yet?
Thats the reason perhaps why my plugins do not show in pro tools.

Would it be possible to run intel version of pro tools on m1 and also the audiogridder to revert back to rosetta and be able to use audiogridder on m1 pro tools?

**Reply by Jarrodsz** [reply]
So m1 is not supported yet?
Thats the reason perhaps why my plugins do not show in pro tools.

Would it be possible to run intel version of pro tools on m1 and also the audiogridder to revert back to rosetta and be able to use audiogridder on m1 pro tools?

**Comment by apohl79** [comment]
1.2 beta has M1 builds of the AAX plugins as well as universal binaries. I think the latest pro tools is supposed to run natively on M1 macs (not rosetta) and should be able to load those plugins. But I can't test them though... I don't have M1 macs nor the latest pro tools version.

**Reply by Jarrodsz** [reply]
Tested them today again pro tools 2021 latest.
They do bot show up.

is there a tool to test pro tools manually tahat you know of? Would be helpfull if there would be a way to validate them against protools instead of pro tools not loading them.

strange thing is they are not in invalid plugins folder either.

you can testdrive protools for free creating a avid.com account then do a trail for pro tools mac or windows

**Comment by apohl79** [comment]
I just checked the avid website (https://avid.secure.force.com/pkb/articles/compatibility/macOS-Big-Sur-Support) and it looks like Pro Tools is still x86 based, as it says "* Designates Apple Silicon (M1) support via Rosetta 2.". That means the AAX arm builds are useless still I guess.
